service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/OrdersNeededAboutToSubmitCallbackController.java

```diff
@@ -55,6 +71,23 @@ public AboutToStartOrSubmitCallbackResponse handleAboutToStartEvent(
             data.put("otherOrderType", "NO");
         }
 
+        Court selectedCourt = getCourtSelection(courtID);
+
+        if (Objects.nonNull(selectedCourt)) {
+            data.remove("court");
```

Comment:
data.put overrides the field if present so can skip this (same for a couple lines down)